TrackingController.java

```diff
@@ -0,0 +1,50 @@
+package de.telran.controller;
+
+import de.telran.dto.CustomerDTO;
+import de.telran.dto.ShipmentDTO;
+import de.telran.dto.TrackingDTO;
+import de.telran.entity.Customer;
+import de.telran.entity.Shipment;
+import de.telran.entity.Tracking;
+import de.telran.service.TrackingService;
+import org.modelmapper.ModelMapper;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+public class TrackingController {
+
+    private TrackingService service;
+    ModelMapper modelMapper = new ModelMapper();
```

Comment:
This should also be private. And it should not be initialized in the class, but rather by using IoC, for instance, by creating a special `@Configuration` class, where the Bean for the ModelMapper is created, but only autowired to the TrackingController.